server/app.js

```diff
@@ -1,6 +1,6 @@
 const express = require('express');
-require('express-async-errors');
-
+//require('express-async-errors');
```

Comment:
Please uncomment this line. We need this so async errors are handled by the middleware.